xapian-applications/omega/templates/query

```diff
@@ -49,17 +50,18 @@ document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
 // -->
 </script>
 </head>
+
 <body bgcolor="#ffffff">
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
 <center>
+<fieldset>
```

Comment:
This fieldset doesn't contain all the fields in the form, so it'd be good to have an `id` or similar for this to make it clear why you're grouping _these_ fields together.